sphinx/writers/latex.py

```diff
@@ -399,6 +401,15 @@ def __init__(self, document, builder):
         if self.elements['polyglossia']:
             self.elements['babel'] = ''  # disable babel forcely
             self.elements['multilingual'] = self.elements['polyglossia']
+            self.elements['classoptions'] += ',' + self.babel.get_language()
+            if builder.config.language:
```

Comment:
Yes. For the babel package, latex writer always give a language option through documentclass even if `language` is not set. But, for the polyglossia, no `\setmainglanguage` is given if `language` is not set.

Of course, it's okay if polyglossia works fine without no main-language.